en/08_Changelogs/6.2.0.md

```diff
@@ -94,6 +96,16 @@ The indicator appears as a "notice" after a configurable initial period (default
 
 See [Unsaved changes indicator](/developer_guides/customising_the_admin_interface/unsaved-changes-indicator/) instructions on how to configure the indicator.
 
+### Move elemental blocks
+
+In the [dnadesign/silverstripe-elemental](/optional_features/elemental/) module, functionality has been added to move elemental blocks from one parent to another, including from one elemental area to another on the same parent record.
+
+![move block button](./_images/move-block-button.png)
+
+The move form provides a lot of flexibility for various configurations, and fields will be hidden if there's only one option to select from. For example, if there's only one elemental area relation on your selected parent record, no dropdown to select an elemental area will be displayed - but if there's multiple, a dropdown will be there for you.
+
+![move block form modal](./_images/move-block-modal.png)
```

Comment:
```suggestion
The [dnadesign/silverstripe-elemental](/optional_features/elemental/) module now includes functionality that allows elemental blocks to be moved from one parent record to another. This includes moving blocks between different elemental areas on the same parent record.

![move block button](./_images/move-block-button.png)

The Move Block Form has been designed for flexibility and will automatically hide fields if there is only one option available. For example:
- If your selected parent record has only one elemental area relation, the dropdown for selecting an elemental area will not be displayed.
- If there are multiple elemental area relations, the dropdown will be available.

![move block form modal](./_images/move-block-modal.png)
```

Reply:
In future please suggest each paragraph's changes separately. You can see GitHub doesn't show a good diff when suggesting changes across multiple paragraphs at once, which means I have to pull the changes into a separate diff viewer.

I'm not doing "Move Block Form" because there's nowhere else it's referred to that way, so using capitals like that is weird.